### PR TITLE
assignment-1 submission bhatipradeep

### PR DESCRIPTION
**Pull request by bhatipradeep**

### CODE REVIEW

File: Submissions/bhatipradeep/Assignment-1/solution.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+class blockchain():
+
+	def __init__(self):
+		self.chain = [self.createGenesis()]
+
+	def createGenesis(self):
+		return block(time.ctime(), "Genesis Block", "00000")
+
+	def mineBlock(self, data):
+	    node = block(time.ctime(),data,self.chain[len(self.chain)-1].hash)
```
Comment (johnsoncarl) [inline Submissions/bhatipradeep/Assignment-1/solution.py:25]: this is correct!!!

File: Submissions/bhatipradeep/Assignment-1/solution.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+class block():
+	def __init__(self, timestamp, data, previousHash = ''):
+		self.timestamp = timestamp
+		self.data = data
+		self.previousHash = previousHash
+		self.hash = self.calculateHash()
+
+	def calculateHash(self):
+	    return sha256(self.data.encode()).hexdigest()
```
Comment (johnsoncarl) [inline Submissions/bhatipradeep/Assignment-1/solution.py]: You missed adding timestamp and Previoushash

**Review by johnsoncarl — CHANGES_REQUESTED** [review]

**Comment by bhatipradeep** [comment]
I already made the changes and made second pull request.

**Review by johnsoncarl — APPROVED** [review]
done 👍

**Pull request merged**